scripts/stablex/request_withdraw.js

```diff
@@ -17,10 +17,6 @@ const argv = require("yargs")
 
 module.exports = async (callback) => {
   try {
-    if (!argv.accountId || !argv.tokenId || !argv.amount) {
```

Comment:
Where did this stuff go? Was it replaced somewhere? Is it not necessary here?

Reply:
This is asserted by the `require('yargs')` statement at the top of the file